Provide selected DACs and datasets

vinjana opened the pull request:
Data is provided as JavaScript objects with datasets nested it DAC.datasets array.

Note that I added lodash as dependency. I got fed up of procedural-style iteration. It's small and  provides `_.cloneDeep(object)` and some functional-style API to collections.

KochTobi commented on js/data.js:
```diff
@@ -26,19 +26,78 @@ let support = {
 };
 
-let studyTableView = {
+class StudyTable {
```
Ah very good. Makes it way more readable!

vinjana commented on _includes/data/access.html:
```diff
@@ -50,9 +56,21 @@ <h2>Request Access</h2>
 			        </div>
 
-					<input type="button" name="sendButton" value="Send Mail" class="btn btn-default" onclick="test()">
+    					<input type="button" name="sendButton" value="Send Mail" class="btn btn-default" onclick="test()">
```
Good question. I am used to 4 spaces. But I am also fine with 2. Problem here was that my editor at home (Atom) is different than the one I use at work (IDEA).

vinjana commented:
I changed the indents to 4. The printing of contacts is also fixed. 
AFAICT this is in a useable state now and you (already) have approved. Do you want to have another look at the current state?

vinjana commented:
Note that you can now access the website on the interims-URL [here](https://ghga.dkfz.de).

KochTobi commented:
The interims URL lacks stylesheets for me though.
Probably not related to this PR

vinjana commented:
Strange. I tested it with Vivaldi/Chrome and Firefox and everything is fine. Could you try figuring out what's the problem? Maybe on the developer console in the browser?

KochTobi commented:
https://ghga.dkfz.de/ works now but the google maps is still in there for some reason. Should not be the case on the master branch.
